setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-with open('kayako/README.txt') as readme:
+with open('src/kayako/README.txt') as readme:
```

Comment:
Had to change this, otherwise setup.py couldn't find the README. I also had to move the `setup.py` to the root directory, otherwise the pip install would crash with an error that no `setup.py` file was present.